src/PKSim.Presentation/Presenters/Charts/ChartPresenter.cs

```diff
@@ -116,6 +118,11 @@ protected virtual void UpdateAnalysisBasedOn(IndividualSimulation simulation, Da
          }
       }
 
+      protected override string NameForColumn(DataColumn dataColumn)
+      {
+         return _chartPresenterContext.CurveNamer.CurveNameForColumn(SimulationFor(dataColumn), dataColumn, _useSimulationNameToCreateCurveName);
```

Comment:
it was not possible to override that behavior. Issue with some refactoring probably. Flag is true for comparison and false for individual simulation